src/test/java/com/microsoft/sqlserver/jdbc/connection/NativeMSSQLDataSourceTest.java

```diff
@@ -0,0 +1,123 @@
+package com.microsoft.sqlserver.jdbc.connection;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutput;
+import java.io.ObjectOutputStream;
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+
+import com.microsoft.sqlserver.jdbc.SQLServerConnectionPoolDataSource;
+import com.microsoft.sqlserver.jdbc.SQLServerDataSource;
+import com.microsoft.sqlserver.jdbc.SQLServerException;
+import com.microsoft.sqlserver.jdbc.SQLServerXADataSource;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+
+@RunWith(JUnitPlatform.class)
+public class NativeMSSQLDataSourceTest extends AbstractTest {
+
+	@Test
+	public void testNativeMSSQLDataSource() throws SQLException {
+		SQLServerXADataSource ds = new SQLServerXADataSource();
+		ds.setLastUpdateCount(true);
+		assertTrue(true == ds.getLastUpdateCount());
```

Comment:
`assertTrue(ds.getLastUpdateCount());` will suffice the purpose. 

No need to compare `true == ds.getLastUpdateCount()); `